CONTRIBUTING.md

```diff
@@ -219,6 +219,7 @@ Note:
 
 * Preferably, these outputs should not be directores but files. For example, if a tool outputs a directory `foo/` containing files `foo/bar.txt` and `foo/baz.txt`, there should be two output arguments `--bar` and `--baz` (as opposed to one output argument which outputs the whole `foo/` directory).
```

Comment:
What if the tool generates a variable number of output files? Depending on input and run configuration? In that case, it makes more sense to output to a directory IMO.

The disadvantage is that no check can be performed on the existence of the output files, but if we don't know what files will be created in the first place that's impossible anyway.

Reply:
I want to avoid that downstream workflows need to perform introspection in a directory to know what they can do with it.

I think you can get very far with optional outputs and outputs with `multiple: true`.

In the very few situations where the directory itself is a recognized standard, we can always output individual files and also the directory as a whole -- that way the downstream user can pick which outputs are relevant to them.